src/persistentStore.js

```diff
@@ -47,3 +47,10 @@ export const persistIsFavorite = (serialNumber, value) =>
     store.set(`${serialNumber}.fav`, value);
 export const getPersistedIsFavorite = serialNumber =>
     store.get(`${serialNumber}.fav`, false);
+
+export const persistIsSendingUsageData = value =>
+    store.set('isSendingUsageData', value);
+export const getIsSendingUsageData = () =>
+    store.get('isSendingUsageData', undefined);
```

Comment:
You could use an explicit default value, false in this case, which will behave the same way as undefined, but later in the debug it looks like false instead of undefined.